app/components/landing-page/GetToKnowUs/index.tsx

```diff
@@ -5,8 +5,8 @@ import GetToKnowUsCard from './GetToKnowUsCard'
 
 const GetToKnowUs = () => {
 	return (
-		<section className='h-full text-white lg:px-14'>
-			<h2 className='my-24 text-center 2sm:text-left'>
+		<section className='h-full text-white mx-auto max-w-9/10'>
+			<h2 className='my-24 text-center 2md:text-left'>
```

Comment:
what the "2" preceding "md:text-left" do?

Reply:
'2md': '880px', at screen >= 880px the text will be left aligned and < 800px will be center aligned.